kythe/go/serving/xrefs/xrefs.go

```diff
@@ -472,12 +472,11 @@ func (t *Table) CrossReferences(ctx context.Context, req *xpb.CrossReferencesReq
 
 	relatedKinds := stringset.New(req.RelatedNodeKind...)
 
-	wantMoreCrossRefs := edgesPageToken == "" &&
```

Comment:
Was this an existing bug, or obsoleted by other changes?

Reply:
Wasn't a bug, but it was vestigial and obsolete.